libs/qCC_glWindow/src/ccGLWindowInterface.cpp

```diff
@@ -4994,7 +4997,10 @@ void ccGLWindowInterface::draw3D(CC_DRAW_CONTEXT& CONTEXT, RenderingParams& rend
 		&& (!renderingParams.useFBO || !m_activeGLFilter))
 	{
 		setStandardOrthoCenter();
+		glFunc->glPushAttrib(GL_DEPTH_BUFFER_BIT);
```

Comment:
And here again you are changing the behavior for everyone. This cross has always been displayed this way for more than 15 years... Maybe add an option to control this behavior? (that's heavier, but you can see how the 'show middle screen cross option' display option is managed currently, and duplicate this behavior).